modules/beta-autopilot-public-cluster/README.md

```diff
@@ -62,125 +62,6 @@ Then perform the following commands on the root folder:
 - `terraform destroy` to destroy the built infrastructure
 
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-## Inputs
```

Comment:
Thanks @DrFaust92! - Something strange with the docs, can you please build again.

Reply:
fixed